Calendars/Calendars.Plugin.Android/CalendarsImplementation.cs

```diff
@@ -456,7 +457,7 @@ public async Task<bool> DeleteCalendarAsync(Calendar calendar)
                 throw new ArgumentException("Cannot delete calendar (probably because it's non-local)", nameof(calendar));
             }
 
-            return await Task.Run(() => Delete(_calendarsUri, long.Parse(calendar.ExternalID))).ConfigureAwait(false);
+            return await Task.Run(() => Delete(_calendarsUri, long.Parse(calendar.ExternalID ?? String.Empty))).ConfigureAwait(false);
```

Comment:
Hm, possibly this should just return false if `ExternalID` is `null` rather than bother to call `Delete`.... but that can be out-of-scope for this PR

Reply:
This was just a "dirty fix" for a more-or-less wrong nullable-detection, as it will never get to this place if the id would be null (except maybe some race-conditions). Because `existing` would be null in that case and it would return false a few lines above...

Maybe a check at the beginning of that method would make sense?

Reply:
A check at the beginning makes sense to me. And yeah I understand and agree with your reasoning but the "dirty fix" still looks a bit odd to me